docs/README.md

```diff
@@ -1,47 +1,43 @@
-# Overview
-**InsertionsClient** updates the versions of NuGet packages in _default.config_ with the corresponding versions specified in _manifest.json_ assets. It also updates the values of properties defined in .props files.
-
-## How does **InsertionsClient** work?
-1. Loads into memory the contents of both _default.config_ and _manifest.json_ as well as all the _.packageconfig_ files listed in the _default.config_
-1. Searches _default.config_ and _.packageconfig_ files for corresponding NuGet packages for each _manifest.json_ asset
-1. For every match, the NuGet version in the config file is updated with that of the corresponding _manifest.json_ asset
-1. The updated _default.config_ and _.packageconfig_'s are saved on disk
+# InsertionsClient
+The **InsertionsClient**, .NET core commnand line executable, updates _NuGet_ packages in files relevant to SDK insertions, such as 
+_default.config_ and .props, with corresponding _manifest.json_ versions.
+
+**Table of Contents**
+- [How it works](#howitworks)
+-  &nbsp;&nbsp;[Input](#input)
+- &nbsp;&nbsp;[Log](#log)
+- &nbsp;&nbsp;[Use](#use)
+- &nbsp;&nbsp;[Output](#output)
+- [Code Base](#codebase)
+- &nbsp;&nbsp;[Microsoft.DotNet.InsertionsClient.Core](#corelib)
+- &nbsp;&nbsp;&nbsp;&nbsp;[Examples](#corelibexamples)
+- &nbsp;&nbsp;[Microsoft.DotNet.InsertionsClient.Console](#console)
+-  &nbsp;&nbsp;&nbsp;&nbsp;[Examples](#consoleexample)
+
+## <a href="#howitworks">How it works</a>
+The **InsertionsClient**...
+1. Iterates through _default.config_ and _.packageconfig_ packages searching for corresponding NuGet versions in _manifest.json_ 
+1. Matching NuGet packages are updated with _manifest.json_ versions 
+1. The updated _default.config_ and _.packageconfig_ files are saved on disk
 1. If an access token is specified with **-a:** switch, binaries for the updated packages are downloaded. Content of the packages are used to update the values of properties defined in `PackagePreprocessorDefinitions` tag in .props files

```

Comment:
Content of the packages _is_ used

Reply:
Updated as "Content**S** of packages are"